src/lib/x509/ocsp.h

```diff
@@ -63,6 +63,16 @@ class BOTAN_PUBLIC_API(2,0) Request final
       CertID m_certid;
    };
 
+/**
+* OCSP response status.
+*
+* see https://tools.ietf.org/html/rfc6960#section-4.2.1
+*/
+enum class BOTAN_PUBLIC_API(2,12) Response_Status_Code
+{
+   SUCCESSFUL = 0, MALFORMED_REQUEST = 1, INTERNAL_ERROR = 2, TRY_LATER = 3, SIG_REQUIRED = 5, UNAUTHORIZED = 6
```

Comment:
Put each enum member on its own line.

For `enum class` it seems cleaner to use a more natural case since unlike old style `enums` they are not all dropped into the global namespace. So eg `Successful`, `Malformed_Request`, ...

Also, style nit with enum we put first `{` on same line as `enum`